sdk/lib/core/enum.dart

```diff
@@ -10,6 +10,9 @@ part of dart.core;
 /// introduced using an `enum` declaration.
 /// Non-platform classes cannot extend or mix in this class.
 /// Concrete classes cannot implement the interface.
+///
+/// The string name of a value is available via the `name` property                                                                                                                    
```

Comment:
I'd go with 
```dart
/// The intentifier used to name an `enum` value is available as a `String`, 
/// via the [`name`](EnumName.name) extension property on the `enum` value.
```
(If that's thow you link to other members without their full name in DartDoc. Where is the DartDoc syntax specification?)

Reply:
I'm not sure if that works, but I'm happy to change it to that if you would prefer. Let me know. :-)